docs/source/tutorials/mantid_basic_course/fitting_data/03_fit_model_choices.rst

```diff
@@ -18,9 +18,9 @@ Multiple peaks + background
 
 It is possible to fit overlapping peaks on a common
 background. Load the file `GEM38370_Focussed` and plot spectrum number 6.
-Zoom in on the 3 peaks around 2.15 Angstroms. Add 3 Gaussian peaks and a Linear Background.
-As can be seen in the screenshot below, the peaks don't have to be of the same type. Run the Fit
-and inspect the results.
+Zoom in on the 3 peaks around 2.15 Angstroms. Add a Linear Background, add a Lorentzian peak
+to the large central peak, and add a Gaussian peak to both of the smaller peaks., as below.
```

Comment:
I think there is a typo in this line

Reply:
I assume @robertapplin means the `.,` as the typo?